src/main/java/hudson/plugins/plot/PlotReport.java

```diff
@@ -191,7 +196,7 @@ public boolean getDisplayTableFlag(int i) {
 
     private Plot getPlot(int i) {
         Plot p = plots.get(i);
-        p.setProject(project);
+        p.setJob(project);
```

Comment:
👍  Also might be worth doing a search from your IDE to see if the setProject is being internally invoked elsewhere and replace with setJob if so.

Reply:
Checked, only in 1 `MatrixPlotPublisher` and can't do it there. No more places.